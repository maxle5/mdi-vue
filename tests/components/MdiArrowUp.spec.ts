
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiArrowUp from "../../src/components/MdiArrowUp.vue";

test("MdiArrowUp snapshot", () => {
  const wrapper = mount(MdiArrowUp, {});
  expect(wrapper.html()).toMatchSnapshot();
});
