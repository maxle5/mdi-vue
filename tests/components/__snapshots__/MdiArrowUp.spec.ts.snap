// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiArrowUp snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M13,20H11V8L5.5,13.5L4.08,12.08L12,4.16L19.92,12.08L18.5,13.5L13,8V20Z" style="fill: currentColor;"></path>
</svg>"
`;
